### PR TITLE
Remove lines/columns from the default environment

### LINKED ISSUE #2543: Weird Drawing of nano
`nano` draws... weirdly under xonsh 0.6.0, but fine under bash.

### PR DESCRIPTION
**Pull request by astronouth7303**
Fixes #2543.

### CODE REVIEW

File: news/2543-lines-columns.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+**Changed:** None
+
+**Deprecated:** None
+
+**Removed:** None
+
+**Fixed:**
+
+* Clean out ``$LINES`` and ``$COLUMNS`` if set, preventing some programs from drawing weird.
```
Comment (scopatz) [inline news/2543-lines-columns.rst]: *weirdly :)

**Comment by scopatz** [comment]
LGTM! Thanks @astronouth7303!

File: news/2543-lines-columns.rst (hunk 1 above)
Comment (astronouth7303) [reply news/2543-lines-columns.rst]: You spend too much time editing papers. 😛

Comment (scopatz) [reply news/2543-lines-columns.rst]: Hahah tell me about it :)

**Comment by scopatz** [comment]
Just some timeout failures.  Thanks again @astronouth7303

**Pull request merged**